int32 box_type

--------------------------

string pallet_name
geometry_msgs/Pose pallet_pose
geometry_msgs/Pose pallet_dim
geometry_msgs/Pose box_dim
geometry_msgs/Pose approaching_pose_1
geometry_msgs/Pose approaching_pose_2
bool box_detected
